MetadataMigration/src/main/java/org/opensearch/migrations/MigrateOrEvaluateArgs.java

```diff
@@ -89,4 +102,11 @@ public String getTransformerConfigParameterArgPrefix() {
                 description = "Path to the JSON configuration file of metadata transformers.")
         private String transformerConfigFile;
     }
+
+    static class MultiTypeResolutionBehaviorConverter implements IStringConverter<IndexMappingTypeRemoval.MultiTypeResolutionBehavior> {
```

Comment:
Don't think you need this class, AFAIK enums are automatically handled

Reply:
Removed, verified it's working without